.github/workflows/release-from-tag.yml

```diff
@@ -0,0 +1,70 @@
+name: Release from GitHub Release
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
+    steps:
+      - name: Get release version from tag
+        id: get_version
+        run: |
+          RELEASE_VERSION=${GITHUB_REF#refs/tags/}
+          echo "Extracted version: $RELEASE_VERSION"
+          # Validate semver format (MAJOR.MINOR.PATCH)
+          echo "$RELEASE_VERSION" | grep -P '^2\.[0-9]+\.[0-9]+$'
+          echo "release_version=$RELEASE_VERSION" >> $GITHUB_OUTPUT
+
+      - uses: actions/checkout@v4
+      - name: Set up JDK 17
```

Comment:
Maybe leave version out of the name, just becomes "one more thing" to keep sync'd